src/games/utility.js

```diff
@@ -1,14 +1,46 @@
-function randomInteger(min, max) {
+const prompt = require('prompt-sync')({ sigint: true });
+
+function randomInteger(min = 1, max = 100) {
   // случайное число от min до (max+1)
   const rand = min + Math.random() * (max + 1 - min);
   return Math.floor(rand);
 }
-function audit(result, version) {
-  if (version !== result) {
+function checkTheResult(correctAnswer, playerVersion) {
+  if (playerVersion !== correctAnswer) {
     console.log('Game over');
     return false;
   }
   console.log('Correct!');
   return true;
 }
-module.exports = { randomInteger, audit };
+function startTheGame(functionPerformed, debug) {
+  let nextOne = true;
+  let numberOfVictories = 3;
+  while (numberOfVictories) {
+    if (!nextOne) {
+      break;
+    }
+    const resultGame = functionPerformed();
+    if (debug) {
+      console.log(`Answer - ${resultGame}`);
+    }
+    const gamerAnswer = prompt('Your answer: ');
+    if (typeof resultGame === 'boolean') {
```

Comment:
You didn't really changed anything from the last time. It is still strangely coupled to `functionPerformed` implementation. It would be great this function could know noting about return values, name of a function, number of parameters, etc.
Or this could be checked by `checkTheResult` function. Anyway this code doesn't belong here.

You can leave it as it is btw